app/services/hyrax/collection_size_service.rb

```diff
@@ -1,40 +0,0 @@
-# returns a list of solr documents for collections the item is a part of
-module Hyrax
-  class CollectionSizeService
```

Comment:
I like that you have an eye out for technical debt that is not in use.  In this case, the service should be deprecated instead of removed.  It has been around for 3 years and it is possible that sites are making use of it.  I would deprecate it with a targeted removal of 3.0.  That should give people plenty of time to comment if they want to keep it around.  

Don't we have a method somewhere else that does this count that we use in search results and other places?  The deprecation notice can point them to the other method.

Reply:
I haven't found anything regarding collection size. The size attribute in Solr is always  empty. If this worked before Collections Sprint, it wasn't actually part of Hyrax. But it doesn't work now. If we want to keep it, it should probably be fixed.